tests/e2e/pgbouncer_types_test.go

```diff
@@ -22,7 +22,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
 
-	"github.com/cloudnative-pg/cloudnative-pg/pkg/specs/pgbouncer"
+	utils2 "github.com/cloudnative-pg/cloudnative-pg/pkg/utils"
```

Comment:
definitely a non blocker nit: alias this one to pkgutils too.

Reply:
Thank you!